git-providers.yaml

```diff
@@ -3,6 +3,10 @@
   apiUrl: https://api.github.com
   repositoryUrl: https://github.com
   type: GITHUB
+  serverProperties:
+    oauthUrl: https://github.com/login/oauth/access_token
+    clientId:
```

Comment:
I am confused, is thie client ID set as an ENV or here?

Reply:
client now loads these settings and no longer uses the env vars